### DISCUSSION TITLE
History Record

### DISCUSSION THREAD

**Discussion by omodia** [mienaiyami/yomikiru#68] [General]
[Issue #69.](https://github.com/mienaiyami/yomikiru/issues/69)

**Comment by omodia** [comment]
Can you provide an update after resolving #64 and #69, before #67?
Because it might take time to resolve #67 and I am a little conflicted about it.

**Reply by mienaiyami** [reply]
#67 wont take much time. Its the opposite, #69 will take a lot more time.
Im sorry but i dont think i might be able to release that by next week. Im also planning to fix up code mess as i started with only basic knowledge when i made this app years ago. I keep seeing some mistakes that might not cause much problem in front but affect performance.

**Reply by omodia** [reply]
> Im sorry but i dont think i might be able to release that by next week.

It's okay. I wanted to have a sort of checkpoint for the app before new image settings are introduced.
